### PR TITLE
perf(robot-server, app): remove commands from GET /runs/:run_id

### PR DESCRIPTION
**Pull request by mcous**
## Overview

This PR removes the unbounded `commands` list from the `GET /runs/:run_id` response in favor of requiring the client to hit `GET /runs/:run_id/commands`. This has the following benefits:

- Requests to fetch simple things like run status, actions list, etc. become much lighter
- We're better set up to add bounds to the returned commands list since it exists as its own endpoint

**WIP, client-side support ongoing**

## Changelog

- remove commands from GET /runs/:run_id responses
- add timestamps to command summary responses
- TBD

## Review requests

TBD

## Risk assessment
TBD

### CODE REVIEW

File: app/src/organisms/Devices/hooks.ts [hunk 1]
```diff
@@ -82,11 +82,10 @@ export function useLights(
 }
 
 export function useIsProtocolRunning(): boolean {
-  const runRecord = useCurrentRun()
+  const runStatus = useRunStatus()
```
Comment (brenthagen) [inline app/src/organisms/Devices/hooks.ts:85]: good catch

**Review by b-cooper — APPROVED** [review]
Looks good

File: app/src/organisms/RunTimeControl/hooks.ts [hunk 2]
```diff
@@ -19,7 +16,14 @@ import {
...
 } from '@opentrons/react-api-client'
 
-import { useCloneRun, useCurrentRun } from '../ProtocolUpload/hooks'
+import {
+  useCloneRun,
+  useCurrentRun,
+  useCurrentRunId,
+  useCurrentRunCommands,
+} from '../ProtocolUpload/hooks'
+import { QueryOptions } from '@testing-library/react'
```
Comment (brenthagen) [inline app/src/organisms/RunTimeControl/hooks.ts]: i think you mean `import { UseQueryOptions } from 'react-query'`?

File: app/src/organisms/RunTimeControl/hooks.ts [hunk 3]
```diff
@@ -62,16 +64,18 @@ export function useRunControls(): RunControls {
...
 }
 
-export function useRunStatus(): RunStatus | null {
-  const runRecord = useCurrentRun()
-
-  const currentRunId = runRecord?.data?.id
+const DEFAULT_STATUS_REFETCH_INTERVAL = 10000 // 10 seconds
+// TODO: remove refetch interval, and pass through optional options param,
+// get runStartTime from top level timestamp
+export function useRunStatus(options?: QueryOptions): RunStatus | null {
```
Comment (brenthagen) [inline app/src/organisms/RunTimeControl/hooks.ts]: and `UseQueryOptions<Run>`

**Review by brenthagen — APPROVED** [review]

**Pull request merged**